AdyenCard/Components/Card/CardComponentConfiguration.swift

```diff
@@ -95,7 +107,7 @@ extension CardComponent {
         }
 
         internal func showAdditionalAuthenticationFields(for issuingCountryCode: String?) -> Bool {
-            self.showsKoreanAuthentication && issuingCountryCode == "KR"
+            self.showsKoreanAuthentication != .hide && issuingCountryCode == "KR"
```

Comment:
no need to use `self...` here i think. i'm not sure why does `FieldMode` enum has 3 cases. seems like here we only care if it is `.hide` or not. is it used anywhere else?

Reply:
👍